markdownlint.js

```diff
@@ -192,6 +192,7 @@ program
   .option('-q, --quiet', 'do not write issues to STDOUT')
   .option('-r, --rules  [file|directory|glob|package]', 'include custom rule files', concatArray, [])
   .option('-s, --stdin', 'read from STDIN (does not work with files)')
+  .option('-v, --verbose', 'write filenames to STDOUT')
```

Comment:
"file names"

Reply:
done in 414f8acfa2bec0318816bf8c4a37ff58e1553e8e!